randovania/generator/filler/retcon.py

```diff
@@ -474,21 +472,23 @@ def _calculate_weights_for(
         return _VICTORY_WEIGHT
 
     potential_uncollected = UncollectedState.from_reach(potential_reach) - current_uncollected
-    if debug.debug_level() > 2:
+    if debug.debug_level() > 1:
         nodes = typing.cast(tuple[Node, ...], potential_reach.game.region_list.all_nodes)
 
         print(f">>> {action}")
         print(f"indices: {potential_uncollected.indices}")
-        print(f"hints: {[hint.as_string for hint in potential_uncollected.hints]}")
         print(f"events: {[event.long_name for event in potential_uncollected.events]}")
+        print(f"hints: {[hint.as_string for hint in potential_uncollected.hints]}")
         print(f"nodes: {[nodes[n].identifier.as_string for n in potential_uncollected.nodes]}")
         print()
 
+    generator = potential_reach.game.game.generator
```

Comment:
I usually put things in `FillerConfiguration`, though I guess that's for preset parameters. I'm thinking maybe there should be a dataclass with all the weights, have the default values be defined directly in that class and store an instance of that class in `GameGenerator`.

Reply:
Addressed.